src/main.jsx

```diff
@@ -112,13 +115,19 @@ const router = createBrowserRouter([
             path: "/termsandconditions",
             element: <TermsAndConditions />,
             },
+
+            {
+            path: "/helpPage",
+            element: <HelpPage />,
+
            
```

Comment:
{
            path: "/helpPage",
            element: <HelpPage />


here curly bracket is not closed so this causes issue in deployment , hey @inkerton  please rectify it and raise pr

Reply:
in line 122  of  src/main.jsx

Reply:
ohkay my mistake I'll do it right now.